stock_picking_product_assortment/models/stock_move.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Tecnativa - Carlos Roca
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
```

Comment:
I don't understand what you intend to do here

Reply:
Is due to a dynamic field domains